### PR TITLE
Use default appinsights environment variable name

### PR DESCRIPTION
**Pull request by c-w**
As per conversation on Slack: we decide to use a single appinsights account for all services so might as well use the same environment variable name everywhere.

### CODE REVIEW

**Review by erikschlegel — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ TRANSLATION_SERVICE_ACCOUNT_KEY="..."
 FACEBOOK_AUTH_TOKEN="..."
 
 FORTIS_FEATURE_SERVICE_HOST="..."
-FORTIS_SERVICES_APPINSIGHTS_KEY="..."
+APPINSIGHTS_INSTRUMENTATIONKEY="..."
```
Comment (jcjimenez) [inline README.md:21]: I noticed that in `project-fortis-spark` the environment variable is `APPLICATION_INSIGHTS_IKEY`. Do we have a PR for that project as well? Or should we just make this project use that key?

**Review by jcjimenez — APPROVED** [review]
LGTM with question

File: README.md (hunk 1 above)
Comment (c-w) [reply README.md:21]: This is amazing. The AppInsights SDKs are inconsistent between each other. Java uses APPLICATION_INSIGHTS_IKEY and all the others use APPINSIGHTS_INSTRUMENTATIONKEY. The latter is recommended, so I think we can stick with that.

**Pull request merged**